components/StoreUser.coffee

```diff
@@ -17,11 +17,15 @@ downloadAvatar = (avatarUrl, callback) ->
     callback new Error 'Avatar request failed'
   req.send()
 
-cleanUpUrl = (callback) ->
-  regex = new RegExp "(?:\\/)?\\?code=[A-Za-z0-9]+"
```

Comment:
Yep. This one is super-old. Not sure how I missed it when I moved this stuff to middlewares